src/OpenLoco/Windows/CompanyWindow.cpp

```diff
@@ -1942,14 +1942,7 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
                     if (company->currentLoan == 0)
                         return;
 
-                    currency32_t stepSize{};
-                    if (*_clickRepeatTicks < 100)
-                        stepSize = 1000;
-                    else if (*_clickRepeatTicks >= 100)
-                        stepSize = 10000;
-                    else if (*_clickRepeatTicks >= 200)
-                        stepSize = 100000;
-
+                    currency32_t stepSize = 1000 * pow(10, (*_clickRepeatTicks / 100));
```

Comment:
You'll need to include `#include <cmath>` also use `std::pow`.
In general we avoid using floating point calculations but as this is UI this is fine.

Reply:
`pow` is the same as `std::pow`, and `cmath` is already included via another header (no idea which one though). I'll include it here since I think it's better for a file to explicitly include the things it requires.

Reply:
Yeah i know c functions are accessible global but in general they should be accessed via the namespace.